cmd/logs.js

```diff
@@ -36,27 +38,29 @@ function isValidNonZeroInteger(number) {
   return /^\d+$/.test(number);
 }
 
-function logs(command, cb) {
+function logs(argsArray, command, cb) {
   const options = init(command);
 
-  // Validate input parameters
-  if (!isValidTimestamp(options.start)) {
-    return handleActionFailure(new Error(`Logs \'start\' ${LogErrorMessages.INVALID_TIMESTAMP}`), cb);
-  }
-  if (!isValidTimestamp(options.end)) {
-    return handleActionFailure(new Error(`Logs \'end\' ${LogErrorMessages.INVALID_TIMESTAMP}`), cb);
-  }
-  if (!isValidNonZeroInteger(options.number)) {
-    return handleActionFailure(new Error(`Logs \'number\' ${LogErrorMessages.INVALID_NONZEROINT}`), cb);
+  // Handle deprecated logs command params
+  if (argsArray != null && argsArray.length > 0) {
+    return handleActionFailure(new KinveyError('DeprecationError', `Version 1.x ${chalk.whiteBright('[from]')} and ${chalk.whiteBright('[to]')} params have been converted to options. Use ${chalk.blueBright('--from')} and ${chalk.blueBright('--to')} to filter by timestamp instead.`), cb);
```

Comment:
I find it a bit odd that you 'paint' some of the words. It's like you have prior knowledge when you are creating the error that it will be logged. And you shouldn't have because it is up to the `handleActionFailure` function to determine what to do with the error. Also, none of the other `new Error/KinveyError` invocations include colored strings. Is it really necessary here?
If it turns out, it is, then I think we'll have to consider doing some changes during the refactoring. :pensive:

Reply:
I don't think the latter case is far-fetched.

My rationale for using colors here is because of the 'hard-line' deprecation. Typically when deprecating things you give users some 'advance' notice. For example, allow `[to]` and `[from]` params as they'd normally function, but print warnings when they're used. In this case we removed them altogether, without any deprecation warnings, so I wanted to emphasize that these parameters weren't removed, but converted to flags.

My opinion is that in this case the colors are okay, because it's a 'special' case, but more importantly, in a future version the command parameters (including the colorful error message) will be removed altogether (probably when the rewrite occurs). Do you think they're okay given these circumstances? If you still have doubts then I have no issue with removing them either. Just trying to do what's best for users

Reply:
:joy: It haven't occurred to me that we will remove the message, actually. Fair enough